apps/web/src/payload/fields/TextInput.ts

```diff
@@ -0,0 +1,36 @@
+import type { GroupField } from 'payload/types';
+
+function TextInput({
+  name,
+  interfaceName,
+  fields = []
+}: Partial<GroupField> = {}): GroupField {
+  return {
+    name: name || 'textinput',
+    type: 'group',
+    interfaceName: interfaceName || 'TextInputType',
+    fields: [
+      {
+        name: 'placeholder',
+        label: 'Placeholder',
+        type: 'text',
+        required: false
+      },
+      {
+        name: 'helpText',
+        label: 'Help Text',
+        type: 'text',
+        required: false
+      },
+      {
+        name: 'label',
```

Comment:
The typing for these fields needs to be updated in `refract`. The label isn't optional, and the typing in refract is incompatible with payload.

Reply:
@queenOfPentacls can you do a write up of what needs to be updated in the refract repo and fwd it to Lynn. This should be resolved before we merge so that the CMS is functional here.